ScreenEffect.h

```diff
@@ -74,7 +74,7 @@ namespace RPG {
 			int shakeStr; //!< Shake strength value used in "Shake Screen" command (0-9). Can be set higher than the default
 			int shakeSpd; //!< Shake speed value used in "Shake Screen" command (0-9). Can be set higher than the default
 			int shakeOffsetX; //!< The screen offset value during a "Shake Screen" command.
-				int _unknown_60; // shakeOffsetY somehow?
+			int shakeOffsetY; //!< An unused Y offset for a vertical Shake Screen. Affects the screen, but gets reset to 0 every frame.
```

Comment:
This one I was able to confirm by forcing it to some value each frame. When I started a new game, the transition in showed the vertical screen offset.

After that, the vertical shake effect was no longer visible. This is because after dynrpg's `onFrame()` callback, RPG_RT always sets this value back to 0.